README.md

```diff
@@ -1,59 +1,33 @@
-# Balena Weather
+# Balena Weather Station
 
-A Raspberry Pi weather station, implemented as [Balena](https://www.balena.io/) multi-container application.
+A Raspberry Pi weather station, running on a [Balena](https://www.balena.io/) multi-container application where you can add more services. This project is inspired by the [Raspberry Pi weather station](https://projects.raspberrypi.org/en/projects/build-your-own-weather-station/0).
 
-[![balena deploy button](https://www.balena.io/deploy.svg)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/hferentschik/balena-weather)
-
-![Grafana Dash](./images/dash.png)
-
-Raspberry Pi 3                                | Weather station
-:--------------------------------------------:|:-------------------------:
-![Raspberry Pi 3](./images/raspberry_pi.png)  |  ![Weather Station](./images/weather_station.png)
-
-<!-- MarkdownTOC levels="2,3,4" autolink="true" indent="  " -->
-
-- [Balena Weather](#balena-weather)
```

Comment:
Any particular reason you removed the table of content? You don't like it or did you not know how to update it. Personally, I find a table of contents very useful in longer documents. 

Just wondering, if it is just a matter of you not wanting to update it, I can re-add it. I am using the MarkdownTOC so it is all automated. If you have another reason, it would be nice to understand your thinking here.